Configuration_adv.hpp

```diff
@@ -366,24 +369,34 @@
         #define ALT_STEPPER_ACCELERATION 2000
     #endif
 
-    // the Circumference of the AZ rotation. 770mm dia.
-    #define ALT_CIRCUMFERENCE 2419.0f
-
     #ifndef AUTOPA_VERSION
         #define AUTOPA_VERSION 1
     #endif
 
-    #if AUTOPA_VERSION == 1
-        // the ratio of the ALT gearbox for AutoPA V1 (40:3)
-        #define ALT_WORMGEAR_RATIO (40.0f / 3.0f)
+    #ifdef OAM
+        #ifndef ALT_ROD_PITCH
+            #define ALT_ROD_PITCH 1.0  // mm/rev
+        #endif
+        // the Circumference of the AZ rotation. 209.1mm radius.
+        #define ALT_CIRCUMFERENCE 209.1 * 2 * PI
```

Comment:
Should `ALT_CIRCUMFERENCE` allowed to be overridden as well?

Reply:
No, not really, since it is fixed by the OAM design whereas the AZ design varies by tripod

Reply:
Alright I see